app/library/Utils.php

```diff
@@ -25,14 +25,13 @@ class Utils extends Component
      */
     public function fetch($data, $element, $default = '')
     {
-        $return = $default;
         if (true === is_object($data) && true === isset($data->$element)) {
-            $return = $data->$element;
+            return $data->$element;
         } elseif (true === is_array($data) && true === isset($data[$element])) {
-            $return = $data[$element];
+            return $data[$element];
         }
 
-        return $return;
+        return $default;
```

Comment:
Come to think of it, I did not mention this in the blog post. I should add this in the comment of the method